winmake.ps1

```diff
@@ -28,6 +28,37 @@ function Local-Machine {
     Run-Command "./test/tools/build/ginkgo.exe -vv  --tags `"$remotetags`" -timeout=90m --trace --no-color $files pkg/machine/e2e/. "
 }
 
+function Win-SSHProxy {
+    # git is not installed by default on windows
+    # fail if git doesn't exist
+    Get-Command git  -ErrorAction SilentlyContinue  | out-null
+    if(!$?){
+        Write-Host "Git not installed, cannot build Win-SSHProxy"
+        Exit 1
+    }
+    # Update the SHA in Makefile as well
+    $GV_GITURL = "https://github.com/containers/gvisor-tap-vsock.git"
+    $GV_SHA = "db608827124caa71ba411cec8ea959bb942984fe"
```

Comment:
Would you mind adding a FIXME or TODO comment here, reminding me to add this package into the windows image?  I can't see any reason not to have it pre-installed.

Reply:
This is actually used as part of the artifacts we ship (it's included in the client release zip as a bundled helper). Its version is also potentially tied to the podman version (although in practice there isn't much divergence)

Reply:
Yeah my comment was about git.